index.md

```diff
@@ -3,87 +3,115 @@
+In machine learning, automatic differentiation is probably the most widely used paradigm, especially in reverse mode.
+However, each method has its own upsides and tradeoffs, which are detailed in the following papers, along with implementation techniques like "operator overloading" and "source transformation":
+
+> [_Automatic differentiation in machine learning: a survey_](https://jmlr.org/papers/v18/17-468.html), Baydin et al. (2018)
+
+> [_A review of automatic differentiation and its efficient implementation_](https://arxiv.org/abs/1811.05031), Margossian (2019)
 
 ## What is JuliaDiff?
-JuliaDiff is an informal organization which aims to unify and document packages written in [Julia](https://julialang.org) for evaluating derivatives. The technical features of Julia, namely, multiple dispatch, source code via reflection, JIT compilation, and first-class access to expression parsing make implementing and using techniques from automatic differentiation easier than ever before (in our biased opinion).
 
+JuliaDiff is an informal [GitHub organization](https://github.com/JuliaDiff/) which aims to unify and document packages written in [Julia](https://julialang.org) for evaluating derivatives.
+The technical features of Julia[^1] make implementing and using differentiation techniques easier than ever before (in our biased opinion).
+
+Discussions on JuliaDiff and its uses may be directed to the [Julia Discourse forum](https://discourse.julialang.org/).
+The ChainRules project maintains a [list of recommended reading](https://www.juliadiff.org/ChainRulesCore.jl/stable/FAQ.html#Where-can-I-learn-more-about-AD-?) for those after more information.
+The [autodiff.org](http://www.autodiff.org/) site serves as a portal for the academic community, though it is often out of date.
 
 ## The Big List
-This is a big list of Julia Automatic Differentiation (AD) packages and related tooling.
-As you can see there is a lot going on here.
-As with any such big lists it rapidly becomes out-dated.
-When you notice something that is out of date, or just plain wrong, please [submit a PR](https://github.com/JuliaDiff/juliadiff.github.io).
 
-This list aims to be comprehensive in coverage.
+What follows is a big list of Julia differentiation packages and related tooling, last updated in January 2024.
+If you notice something inaccurate or outdated, please [open an issue](https://github.com/JuliaDiff/juliadiff.github.io/issues) to signal it.
```

Comment:
Why the change from submit a PR to open an issue?

Reply:
Because beginners will find it easier to open an issue than to fork, clone, modify, push and PR